samples/net/lwm2m_client/frdm_k64f.overlay

```diff
@@ -5,6 +5,7 @@
  */
 &uart2 {
 	current-speed = <115200>;
+	hw-flow-control = <1>;
```

Comment:
This should just be:

`hw-flow-control`;

Boolean are if the property name exists or not.